src/main/java/com/esri/samples/localserver/local_server_services/LocalServerServicesController.java

```diff
@@ -18,19 +18,31 @@
 
 import java.io.File;
 
-import com.esri.arcgisruntime.localserver.*;
+import com.esri.arcgisruntime.localserver.LocalFeatureService;
+import com.esri.arcgisruntime.localserver.LocalGeoprocessingService;
+import com.esri.arcgisruntime.localserver.LocalMapService;
+import com.esri.arcgisruntime.localserver.LocalServer;
+import com.esri.arcgisruntime.localserver.LocalServerStatus;
+import com.esri.arcgisruntime.localserver.LocalService;
 import com.esri.arcgisruntime.util.ListenableList;
 
 import javafx.application.HostServices;
 import javafx.application.Platform;
 import javafx.fxml.FXML;
-import javafx.scene.control.*;
+import javafx.scene.control.Alert;
 import javafx.scene.control.Alert.AlertType;
+import javafx.scene.control.Button;
+import javafx.scene.control.ComboBox;
+import javafx.scene.control.Label;
+import javafx.scene.control.ListView;
+import javafx.scene.control.TextArea;
 
 public class LocalServerServicesController {
 
   @FXML private Button btnStartServer;
   @FXML private ComboBox<String> serviceOptions;
+  @FXML private Button btnSelectPackage;

```

Comment:
I prefer the naming convention: `selectPackageButton`. Most of our samples use this convention.